sdk/src/assertions/actions.rs

```diff
@@ -194,7 +197,8 @@ impl Action {
         matches!(
             self.software_agent,
             Some(SoftwareAgent::ClaimGeneratorInfo(_))
-        ) || self.changes.is_some() // only defined for v2
+        ) || self.changes.is_some()
+            || self.description.is_some() // only defined for v2
```

Comment:
It is probably time to always generate v2 assertions and not bother with this logic, but I'll update that later